### PR TITLE
insert a link to our manta ray tutorial series

### PR DESCRIPTION
**Pull request by daneklustig**

### CODE REVIEW

File: content/tutorials/jupyter-notebooks.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ It's quite easy to figure out, but if you get stuck, here are some initial steps
 - When asked to select a kernel, select Python 3.
 - To make stuff happen (e.g. to run code cells), use the menus at the top of the JupyterLab console, or at the top of your current Jupyter notebook.
 
+In you prefer a step-by-step tutorial, you can watch our thourough [Manta Ray tutorial mini series] (https://www.youtube.com/playlist?list=PL_dn0wVs9kWqSO2iCXvrWuxKFSgVr0Jrw) on our YouTube channel.
```
Comment (daneklustig) [inline content/tutorials/jupyter-notebooks.md]:
```suggestion
In you prefer a step-by-step tutorial, you can watch our thorough [Manta Ray tutorial mini series] (https://www.youtube.com/playlist?list=PL_dn0wVs9kWqSO2iCXvrWuxKFSgVr0Jrw) on our YouTube channel. 
```

Comment (daneklustig) [reply content/tutorials/jupyter-notebooks.md]:
```suggestion
In you prefer a step-by-step tutorial, you can watch our thourough [Manta Ray tutorial mini series](https://www.youtube.com/playlist?list=PL_dn0wVs9kWqSO2iCXvrWuxKFSgVr0Jrw) on our YouTube channel. 
```

File: content/tutorials/jupyter-notebooks.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ It's quite easy to figure out, but if you get stuck, here are some initial steps
 - When asked to select a kernel, select Python 3.
 - To make stuff happen (e.g. to run code cells), use the menus at the top of the JupyterLab console, or at the top of your current Jupyter notebook.
 
+In you prefer a step-by-step tutorial, you can watch our thorough [Manta Ray tutorial mini series] (https://www.youtube.com/playlist?list=PL_dn0wVs9kWqSO2iCXvrWuxKFSgVr0Jrw) on our YouTube channel.
```
Comment (ttmc) [inline content/tutorials/jupyter-notebooks.md]:
```suggestion
In you prefer a step-by-step tutorial, you can watch our thorough [Manta Ray tutorial mini series](https://www.youtube.com/playlist?list=PL_dn0wVs9kWqSO2iCXvrWuxKFSgVr0Jrw) on our YouTube channel. 
```

Comment (ttmc) [reply content/tutorials/jupyter-notebooks.md]: I'm not sure why this suggestion didn't work earlier. I re-made it.

**Comment by kremalicious** [comment]
Did anybody try if our video plugin also can handle a playlist link here? https://docs.oceanprotocol.com/test/#videos

Would be alright if the video player is simply shown on that page

**Comment by ttmc** [comment]
I did not try and I'm pretty sure Daniel didn't either.

**Comment by kremalicious** [comment]
well, playlist doesn't work to embed but simply adding all videos under each other is also nice to give them more exposure so I did that:

Screen Shot 2019-06-20 at 11 41 21

**Comment by ttmc** [comment]
Very nice, thanks Matthias!

**Pull request merged**